dask/array/tests/test_slicing.py

```diff
@@ -378,3 +379,61 @@ def test_slice_list_then_None():
     y = x[[2, 1]][None]
 
     assert eq(y, np.zeros((1, 2, 5)))
+
+
+class ReturnItem(object):
+    def __getitem__(self, key):
+        return key
+
+
+@skip
```

Comment:
`unittest.skip` doesn't work on bare functions.  I could use either pytest.skip or nose.skip but then I have to choose.